storage/remote/write_test.go

```diff
@@ -381,14 +382,16 @@ func TestWriteStorageApplyConfig_PartialUpdate(t *testing.T) {
 	require.NoError(t, s.Close())
 }
 
-func TestOTLPWriteHandler(t *testing.T) {
+func TestOTLPWriteHandlerNaming(t *testing.T) {
```

Comment:
I don't think the test name should change, since it's intentionally named so as to correspond to the handler under test:
```suggestion
func TestOTLPWriteHandler(t *testing.T) {
```